routes/events/(pages)/organizing/index.tsx

```diff
@@ -34,18 +35,46 @@ export default async function Homepage(req: Request, ctx: AppContext) {
   return (
     <>
       <div className="grid grid-cols-2 gap-8">
-        {events.map((event) => (
-          <div className="rounded-md overflow-hidden">
-            <img
-              src="https://images.unsplash.com/photo-1647123817877-a35e45f66e9c?&auto=format&fit=crop&w=1887&q=80"
-              alt=""
-              class="w-full h-52 object-cover"
-            />
-            <div className="flex flex-col grow px-2">
-              <h3 className="font-semibold text-2xl -translate-y-4 bg-white">{event.value?.name}</h3>
+        {events.map((event) => {
+          const e = event.value;
+          if (!e) {
+            return null;
+          }
+
+          return (
+            <div className="rounded-md overflow-hidden">
+              {e.banner.path ? (
+                (() => {
+                  const url = imageKit.url({
+                    path: e.banner.path,
+                    transformation: [
+                      {
+                        width: "400",
+                        quality: "85",
```

Comment:
Didn't you say we're limited on transformations with imagekit

Reply:
the reason we're using imagekit is bc it's (more or less) unlimited.